tests/python/pytorch/dataloading/test_dataloader.py

```diff
@@ -622,6 +622,26 @@ def test_edge_dataloader_excludes(
             break
 
 
+def dummy_worker_init_fn(worker_id):
+    pass
+
+
+def test_dataloader_worker_init_fn():
```

Comment:
It seems that the test case added here does not cover the fix.

Reply:
It does.  Previously it will throw an error even if `worker_init_fn` is an empty function (as long as it's not None).